modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/factory/GlobalCSSCETImplFactoryImpl.java

```diff
@@ -60,9 +60,13 @@ public void validate(
 		GlobalCSSCET newGlobalCSSCET = new GlobalCSSCETImpl(
 			StringPool.NEW_LINE, newTypeSettingsUnicodeProperties);
 
-		if (!Validator.isUrl(newGlobalCSSCET.getURL())) {
+		String url = newGlobalCSSCET.getURL();
+
+		if (!Validator.isUrl(url)) {
 			throw new ClientExtensionEntryTypeSettingsException(
-				"please-enter-a-valid-url");
+				"Invalid URL: " + url,
+				"url-x-is-invalid", url
```

Comment:
@izaera rightly noted that we shouldn't be changing the language keys with this PR. So, let's keep this as `"please-enter-a-valid-url"`. I confused this by commenting that you should change the language keys in my last review. Sorry for that. I'll create a [separate story](https://liferay.atlassian.net/browse/LPS-200193) to review all the error messages for consistency. Please revert all the changes to the language keys.

I think it's still fine to use `"Invalid URL: " + url` for the plaintext error though.

Reply:
I'm OK with changing the keys, if we all agree and think they suit the situation better. Even if it is outside of the scope of the URL I don't think it's so important to merge it in a separate PR.

@ethib137 I'll let you decide what to do :-)